apps/src/templates/teacherDashboard/sections.js

```diff
@@ -69,6 +70,31 @@ export function renderLoginTypeAndSharingControls(sectionId) {
   );
 }
 
+export function renderSectionTable(sectionId, loginType) {
+  const dataUrl = `/v2/sections/${sectionId}/students`;
+  const element = document.getElementById('student-table-react');
+
+  $.ajax({
+    method: 'GET',
+    url: dataUrl,
+    dataType: 'json'
+  }).done(studentData => {
+    studentData = studentData.map((section) => {
```

Comment:
Each element in this array is a student, not a section, right? (Possible I'm misunderstanding)

Reply:
no you're right, I just missed that one apparently